page-builder/hooks/html.md

```diff
@@ -26,6 +26,36 @@ echo apply_filters( 'siteorigin_panels_before_row', '', $panels_data['grids'][$g
 echo apply_filters( 'siteorigin_panels_after_row', '', $panels_data['grids'][$gi], $grid_attributes );
 ```
 
+
+### Inside Rows Before and After
+///
+
+You're able to output additional content inside of the rows and before/after the cells have been added. 
```

Comment:
How about:

"You can output additional content inside the rows and before/after the cells have been added."